packages/host/src/components/Login/Login.tsx

```diff
@@ -27,18 +33,76 @@ const StoreAutocompleteInput: React.FC<AutocompleteRenderInputParams> =
     );
   };
 
+const storeSorter = (a: Store, b: Store) => {
```

Comment:
the api provides stores in a random ( database insertion? ) order and has no sort option.
I like alphabetical.. so have done this for now and raised an issue for the api team.